newsfragments/43191.improvement.rst

```diff
@@ -1 +0,0 @@
-Bash script files (``.sh`` and ``.bash``) with Jinja templating enabled (without the space after the file extension) are now rendered into a temporary file, and then executed. Instead of being directly executed as inline command.
```

Comment:
I think that one did not make it into 2.10.3 where the PR has been merged and released. Was it not included deliberately? And if not, then shally we update the 2.10.3 to include it?

Reply:
Ya, I saw the origin PR was part of 2.10.3 [release notes](https://github.com/apache/airflow/blob/ee07d6339a3b1a1b4451c20f5109ab51dcd13ebe/RELEASE_NOTES.rst#L55) and I removed it from the 2.10.4 release notes. 

I can create a separate PR for it to be merged into the 2.10.3 release notes. Does that sound good @potiuk?